project/driver_controls/inc/event_arbiter.h

```diff
@@ -0,0 +1,19 @@
+#pragma once
```

Comment:
This module seems to be missing an init function. Also, please write tests for every module. Having an integration-style test where the interactions between modules is great, but focused tests per module are always useful for verifying basic behavior and edge cases.